### PR TITLE
Dashboard: Enforce minimum dashboard refresh interval to 5 seconds per default

### LINKED ISSUE #22493: Dashboard: Enforce a global minimum dashboard refresh interval per default
**What would you like to be added**:
Enforce a global minimum dashboard refresh interval per default, suggested 5 seconds.

**Why is this needed**:
#19416 added support for a global minimum dashboard refresh interval which by default is not set/unrestricted. To make Grafana more "safe" by default we want to introduce this breaking change.

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
Enforce min dashboard refresh interval to 5 seconds per default. 

**Which issue(s) this PR fixes**:
Fixes #22493 

**Special notes for your reviewer**:
Used 5 seconds as default since this is the default in UI.

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

File: pkg/setting/setting.go [hunk 1]
```diff
@@ -768,7 +768,7 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	// read dashboard settings
 	dashboards := iniFile.Section("dashboards")
 	DashboardVersionsToKeep = dashboards.Key("versions_to_keep").MustInt(20)
-	MinRefreshInterval, err = valueAsString(dashboards, "min_refresh_interval", "")
+	MinRefreshInterval, err = valueAsString(dashboards, "min_refresh_interval", "5s")
```
Comment (marefr) [inline pkg/setting/setting.go:771]: May want to change this so that `MinRefreshInterval` becomes a `time.Duration`?

**Review by bergquist — APPROVED** [review]
LGTM

**Review by aknuds1 — APPROVED** [review]
LGTM

File: pkg/setting/setting.go (hunk 1 above)
Comment (bergquist) [reply pkg/setting/setting.go:771]: Sounds good. But that could be another PR. :)

Comment (marefr) [reply pkg/setting/setting.go:771]: Yeah, noticed it quite a bit of changes so probably some time later.

**Pull request merged**